Problem with typing

Some error is occurring with sentry for python (using python 3.8)

```
[ERROR] AttributeError: type object 'Callable' has no attribute '_abc_registry'
Traceback (most recent call last):
  File "/var/task/handler.py", line 609, in lambda_handler
    return LambdaHandler.lambda_handler(event, context)
  File "/var/task/handler.py", line 240, in lambda_handler
    handler = cls()
  File "/var/task/handler.py", line 134, in __init__
    self.app_module = importlib.import_module(self.settings.APP_MODULE)
  File "/var/lang/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/var/task/src/main.py", line 6, in <module>
    import sentry_sdk
  File "/var/task/sentry_sdk/__init__.py", line 1, in <module>
    from sentry_sdk.hub import Hub, init
  File "/var/task/sentry_sdk/hub.py", line 8, in <module>
    from sentry_sdk._compat import with_metaclass
  File "/var/task/sentry_sdk/_compat.py", line 3, in <module>
    from sentry_sdk._types import MYPY
  File "/var/task/sentry_sdk/_types.py", line 2, in <module>
    from typing import TYPE_CHECKING as MYPY
  File "/var/task/typing.py", line 1357, in <module>
    class Callable(extra=collections_abc.Callable, metaclass=CallableMeta):
  File "/var/task/typing.py", line 1005, in __new__
    self._abc_registry = extra._abc_registry

```
Any hint?

--- a/sentry_sdk/api.py
+++ b/sentry_sdk/api.py
@@ -15,7 +15,7 @@ if MYPY:
     from typing import TypeVar
     from typing import ContextManager
 
-    from sentry_sdk._types import Event, Hint, Breadcrumb, BreadcrumbHint
+    from sentry_sdk._types import Event, Hint, Breadcrumb, BreadcrumbHint, ExcInfo
     from sentry_sdk.tracing import Span
 
     T = TypeVar("T")
@@ -93,7 +93,7 @@ def capture_message(
 
 @hubmethod
 def capture_exception(
-    error=None,  # type: Optional[BaseException]
+    error=None,  # type: Optional[Union[BaseException, ExcInfo]]
     scope=None,  # type: Optional[Any]
     **scope_args  # type: Dict[str, Any]
 ):

--- a/sentry_sdk/api.py
+++ b/sentry_sdk/api.py
@@ -14,6 +14,7 @@ if MYPY:
     from typing import Callable
     from typing import TypeVar
     from typing import ContextManager
+    from typing import Union
 
     from sentry_sdk._types import Event, Hint, Breadcrumb, BreadcrumbHint, ExcInfo
     from sentry_sdk.tracing import Span

